### ISSUE TITLE
[Request/Question] waring if breakpoint is not set

### ISSUE THREAD

**Issue by di5abled** [zurb/foundation-sites#8207]
Is there a reason that you just fall back to `0` in your breakpoint mixin instead of displaying a waring or error?

See: https://github.com/zurb/foundation-sites/blob/develop/scss/util/_breakpoint.scss#L66

I would recommend replacing:
```
@else {
  $bp: 0;
}
```

with:
```
@else {
    @error 'Your given breakpoint "#{$val}" is not defined in your settings.';
}
```

**Comment by gakimball** [comment]
We can add a fallback, but when would you be referencing a breakpoint name you haven't configured?

**Comment by di5abled** [comment]
If the warning above is ok I could make a PR if you want!?

**Comment by gakimball** [comment]
Definitely! I'd include the fallback and the error.

**Comment by di5abled** [comment]
Perfekt, then you get the PR this week ;)

**Comment by gakimball** [comment]
Merged #8274
